subprojects/studio-plugin/src/test/groovy/org/gradle/profiler/studio/plugin/StudioPluginIntegrationTest.groovy

```diff
@@ -15,6 +17,10 @@ import static org.gradle.profiler.client.protocol.messages.StudioRequest.StudioR
 import static org.gradle.profiler.client.protocol.messages.StudioSyncRequestCompleted.StudioSyncRequestResult.FAILED
 import static org.gradle.profiler.client.protocol.messages.StudioSyncRequestCompleted.StudioSyncRequestResult.SUCCEEDED
 
+/**
+ * Note: This class doesn't extend Spock Specification, but it extends IntelliJ HeavyPlatformTestCase,
+ * so you should always use `assert` when asserting statements
+ */
```

Comment:
This sounds dangerous for future changes. Shall we drop using Spock altogether (including `when:` and `then:`)?

Reply:
Changed, now we use Spock `Specification`, but we setup Ide via Composition, WDYT?